### ISSUE TITLE
"Goto next node(fold)" not working on filtered map, 2nd try :)

### ISSUE THREAD

**Issue by theelectricsoul** [freeplane/freeplane#962]
### Bug
`Goto (next|previous) node(fold) ` is not working on filtered maps.

### Reproduce
Let me try to explain again. This time the video is fullscreen. 

1. Remove all filters
2. Unfold all nodes
3. Filter node "1"
4. Collapse children nodes
5. Try navigating using command `Goto (next|previous) node(fold) `
6. 🔴Navigating should not work
7. Remove all filters
8. Node "1" should stay collapsed
9. Try navigating again using `Goto (next|previous) node(fold) `
10. 🟢This should have worked

**Conclusion**
Command `Goto (next|previous) node(fold) ` not working on filtered map.

[Video](https://user-images.githubusercontent.com/10260832/213943279-c9a9fe51-c349-4bea-84c7-70c94b8638e0.mp4)

### Files
[Mindmap.zip](https://github.com/freeplane/freeplane/files/10475641/Mindmap.zip)

### Version
1.11.1 alpha

**Comment by theelectricsoul** [comment]
I just noticed that the following commands do not work either, when filter is active.

`Goto (next|previous) node`

**Comment by theelectricsoul** [comment]
Maybe this thought helps. I assume that the following functions do filter the map, even though it says "no filtering" when i use them. 
![image](https://user-images.githubusercontent.com/10260832/214004413-3ef14801-b032-432b-9706-c1579aed766c.png)

If they actually do filter, then let me say; when "Jump in" is active, the navigation is working when it comes to the "Goto commands".

**Comment by dpolivaev** [comment]
Preview 1.11.1_10
